src/WrapperApp/components/InputEditor/InputEditorPanel.tsx

```diff
@@ -49,11 +45,8 @@ export default function InputEditorPanel({
 		async (editorJSON: EditorJson): Promise<SimulationInputFiles> => {
 			await addCustomStoppingPowerTableToEditorJSON(editorJSON);
 
+			//Currently there is only one generator
 			switch (generator) {
-				case 'remote':
```

Comment:
Should I also remove:

- the code responsible for selecting a potentially different converter
- the code responsible for sending a request to a remote converter from src/services/ShSimulatorService.tsx